### ISSUE TITLE
Add daydream support

### ISSUE THREAD

**Issue by rafaellago** [markusfisch/ShaderEditor#44]
I usually charge my phone on a dock. And it has the possibility to activate a selected daydream when it's there... 

Would be awesome to use a shader as a Daydream, just like we add to live wallpaper.

**Comment by markusfisch** [comment]
Yes, that's a good idea and I'd love to add this. What kept me from doing so is that it'd mean to raise the minSdkVersion to API level 17 which is Android 4.2. Currently the app runs on API level 9+ because I didn't want to lock out the old devices (still quite common in developing countries then).

As time goes by, raising minSdkVersion to something more up to date is probably quite reasonable.
